src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user != null && user.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
```suggestion
        return user != null && user.getPassword().equals(password);
```
You can simplify it

Reply:
How can I implement the suggested changes? I've opened this project in the environment of development on my desktop and then I've implemented the suggested changes. After that I've created a new commit. Do I need to create a new pull request or what? Thank you in advance.